src/utils/_mixins.scss

```
@@ -0,0 +1,29 @@
+@mixin circle($size) {
+  height: $size;
+  width: $size;
+  border-radius: 50%;
+}
+
+@mixin dot-size($size) {
+  height: $size;
+  width: $size;
+  border-radius: 50%;
+}
+
+@mixin seconds-clockhand-size {
+  height: $seconds-clockhand-height;
+  width: $seconds-clockhand-width;
+  border-radius: 50%;
+}
+
+@mixin minutes-clockhand-size {
+  height: $minutes-clockhand-height;
+  width: $minutes-clockhand-width;
+  border-radius: 50%;
+}
+
+@mixin hours-clockhand-size {
+  height: $hours-clockhand-height;
+  width: $hours-clockhand-width;
+  border-radius: 50%;
+}
```

this is unnessesary to have so many mixins, you have make one universal mixin with parameters
```suggestion
@mixin clockhand-size($height, $width) {
  height: $height;
  width: $width;
  border-radius: 50%;
}

//Then, just use them in your clockhand classes 

@include clockhand-size($hours-clockhand-height, $hours-clockhand-width);
```